notebooks/MeasurementOrFact_Analysis.ipynb

```diff
@@ -0,0 +1,2951 @@
+{
```

Comment:
<p>I find myself often just scrolling past these tables without looking at them. Maybe doing a <code>df.describe()</code> would be better for some of these?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Thank you for the revew! I will try a <code>df.describe()</code> here and figure out if the findings hold some story. </p>